echopype/tests/test_convert.py

```diff
@@ -14,7 +14,7 @@
 import xarray as xr
 import pytest
 from pathlib import Path
-from ..echodata import open_raw
+from .. import open_raw
```

Comment:
Let's try to be explicit here for tests file. You can simply just do `from echopype import open_raw`.